<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>481</width>
      <height>572</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>PDF a IMG</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>10</y>
        <width>465</width>
        <height>203</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="button_Select_PDF_PDFAIMG">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>111</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>SELECCIONAR ARCHIVO PDF</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>CARPETA DE SALIDA</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_Select_CARPETA_PDFAIMG">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>111</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>CARPETA</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_PROCESAR_PDFAIMG">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>155</y>
         <width>341</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF A IMAGEN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_path_PDF_PDFAIMG">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>45</y>
         <width>318</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_path_CARPETA_PDFAIMG">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>114</y>
         <width>318</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Margen PDF</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>10</y>
        <width>461</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="button_Select_PDF_MARGENPDF">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>111</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>SELECCIONAR ARCHIVO PDF</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_Margen_Superior_MARGENPDF">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>421</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_Margen_Inferior_MARGENPDF">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>421</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_Margen_Derecha_MARGENPDF">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>421</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_Margen_Izquierda_MARGENPDF">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>421</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>88</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>MARGEN SUPERIOR</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>MARGEN INFERIOR</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>MARGEN DERECHA</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>MARGEN IZQUIERDA</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_PROCESAR_MARGENPDF">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>296</y>
         <width>341</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>CORTAR MARGEN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_path_PDF_MARGENPDF">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>45</y>
         <width>318</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_num_margen_izquierda_MARGENPDF">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>240</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_num_margen_derecha_MARGENPDF">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>190</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_num_margen_inferior_MARGENPDF">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>140</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_num_margen_superior_MARGENPDF">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>88</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Eliminar Pag. PDF</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>10</y>
        <width>461</width>
        <height>531</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>SELECCIONAR ARCHIVO PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_Select_PDF_ELIMPAGPDF">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>111</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_Eliminar_Paginas_ELIMPAGPDF">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>496</y>
         <width>341</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>ELIMINAR PAGINAS</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>PAGINAS A ELIMINAR: </string>
       </property>
      </widget>
      <widget class="QTextEdit" name="text_pages_delete_ELIMPAGPDF">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>431</width>
         <height>44</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_path_PDF_ELIMPAGPDF">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>45</y>
         <width>318</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>:</string>
       </property>
      </widget>
      <widget class="QListWidget" name="List_0_Paginas_PDF_ELIMPAGPFD">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>139</width>
         <height>347</height>
        </rect>
       </property>
      </widget>
      <widget class="QListWidget" name="List_1_Paginas_PDF_ELIMPAGPFD">
       <property name="geometry">
        <rect>
         <x>165</x>
         <y>140</y>
         <width>139</width>
         <height>347</height>
        </rect>
       </property>
      </widget>
      <widget class="QListWidget" name="List_2_Paginas_PDF_ELIMPAGPFD">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>140</y>
         <width>139</width>
         <height>347</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
